leveldbstore/store.go

```diff
@@ -0,0 +1,48 @@
+package leveldbstore
+
+import (
+	"github.com/eventials/go-tus"
+	"github.com/mitchellh/go-homedir"
+	"github.com/syndtr/goleveldb/leveldb"
+	"path/filepath"
+)
+
+type LeveldbStore struct {
+	db *leveldb.DB
+}
+
+func NewLeveldbStore() (tus.Store, error) {
+	home, err := homedir.Dir()
```

Comment:
Why not pass the path as an argument? What if I want another location, plus, it removes a random dependency. It also could be replaced by "os/user" package.